src/App.tsx

```diff
@@ -1,14 +1,84 @@
-/* eslint-disable max-len */
-import React from 'react';
+import {
+  FC, useCallback, useEffect, useMemo, useState,
+} from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
-import { TodoModal } from './components/TodoModal';
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
 import { Loader } from './components/Loader';
+import { TodoModal } from './components/TodoModal';
+import { FilterBy } from './types/FilterBy';
+
+const findQueryInTheString = (query: string, title: string) => {
+  const queryToLower = query.toLowerCase().trim();
+  const titleToLower = title.toLowerCase();
+
+  return titleToLower.includes(queryToLower);
+};
+
+const debounce = (f: (string: string) => void, delay: number) => {
+  let timerId: any;
+
+  return (...args: any) => {
```

Comment:
Consider using a more specific type for the arguments, such as Parameters<typeof f> instead of any.